src/main/frontend/pipeline-console-view/pipeline-console/main/NoStageStepsFallback.tsx

```diff
@@ -0,0 +1,63 @@
+import { Suspense, useEffect, useState } from "react";
+
+import {
+  getConsoleBuildOutput,
+  StepInfo,
+  StepLogBufferInfo,
+} from "../../../common/RestClient.tsx";
+import { Result } from "../../../pipeline-graph-view/pipeline-graph/main/PipelineGraphModel.tsx";
+import ConsoleLogStream from "./ConsoleLogStream.tsx";
+
+async function fetchData(): Promise<StepLogBufferInfo> {
+  const consoleBuildOutput = await getConsoleBuildOutput();
+
+  return {
+    lines: consoleBuildOutput?.split("\n") ?? [],
+    startByte: 0,
+    endByte: 0,
+  };
+}
+
+export function NoStageStepsFallback() {
+  const step: StepInfo = {
+    id: "step-1",
+    name: "Step 1",
+    title: "Step 1 Title",
+    state: Result.success,
+    completePercent: 100,
+    type: "STEP",
+    startTimeMillis: 0,
+    totalDurationMillis: 0,
+    stageId: "",
+    pauseDurationMillis: 0,
+  };
+
+  const [logBuffer, setLogBuffer] = useState<StepLogBufferInfo>();
+
+  useEffect(() => {
+    fetchData()
+      .then((data) => {
+        setLogBuffer(data);
+        return data;
+      })
+      .catch((err) => console.log(err));
+  }, []);
+
+  return (
+    <Suspense>
+      <div
+        className={"pgv-stage-steps"}
+        key={`stage-steps-container-full-build}`}
```

Comment:
```suggestion
        key={"stage-steps-container-full-build"}
```
but same as previous